packages/serve/index.ts

```diff
@@ -94,60 +93,64 @@ function serve() {
 		process.stdout.write(
 			"\n" +
 				chalk.bold(
-					"✖ We didn't find any webpack-dev-server dependency in your project,"
+					"✖ We didn't find any webpack-dev-server dependency in your project,",
 				) +
 				"\n" +
 				chalk.bold.green("  'webpack serve'") +
 				" " +
 				chalk.bold("requires you to have it installed ") +
-				"\n\n"
+				"\n\n",
 		);
 		return inquirer
 			.prompt([
 				{
-					type: "confirm",
-					name: "confirmDevserver",
+					default: "Y",
 					message: "Do you want to install it? (default: Y)",
-					default: "Y"
-				}
+					name: "confirmDevserver",
+					type: "confirm",
+				},
 			])
-			.then(answer => {
-				if (answer["confirmDevserver"]) {
+			.then((answer: {
+				confirmDevserver: boolean,
+			}) => {
+				if (answer.confirmDevserver) {
 					return inquirer
 						.prompt(
 							List(
 								"confirmDepType",
 								"What kind of dependency do you want it to be under? (default: devDependency)",
-								["devDependency", "optionalDependency", "dependency"]
-							)
+								["devDependency", "optionalDependency", "dependency"],
+							),
 						)
-						.then(depTypeAns => {
-							const packager = getRootPathModule("package-lock.json")
+						.then((depTypeAns: {
+							confirmDepType: string;
+						}) => {
+							const packager: string = getRootPathModule("package-lock.json")
 								? "npm"
 								: "yarn";
-							let spawnAction;
-							if (depTypeAns["confirmDepType"] === "devDependency") {
+							let spawnAction: () => SpawnSyncReturns<Buffer>;
+							if (depTypeAns.confirmDepType === "devDependency") {
 								if (packager === "yarn") {
-									spawnAction = _ => spawnYarnWithArg("--dev");
+									spawnAction = () => spawnYarnWithArg("--dev");
 								} else {
-									spawnAction = _ => spawnNPMWithArg("--save-dev");
+									spawnAction = () => spawnNPMWithArg("--save-dev");
 								}
 							}
-							if (depTypeAns["confirmDepType"] === "dependency") {
+							if (depTypeAns.confirmDepType === "dependency") {
 								if (packager === "yarn") {
-									spawnAction = _ => spawnYarnWithArg(" ");
+									spawnAction = () => spawnYarnWithArg(" ");
 								} else {
-									spawnAction = _ => spawnNPMWithArg("--save");
+									spawnAction = () => spawnNPMWithArg("--save");
```

Comment:
void

Reply:
This is a promise, hence I didn't add `_: void`. If we add it, we'll have to pass something here or use syntax `_?: void`.
https://github.com/webpack/webpack-cli/blob/617af0bddb5140c07b1a0079b377311b10dd027d/packages/serve/index.js#L150

Reply:
`_?: void` syntax, that's normal convention across the repository.